rest/taskrouter/statistics/workflow/example-1/example-1.5.x.cs

```diff
@@ -0,0 +1,34 @@
+// Download the twilio-csharp library from
+// https://www.twilio.com/docs/libraries/csharp#installation
+using Newtonsoft.Json.Linq;
+using System;
+using Twilio;
+using Twilio.Rest.Taskrouter.V1.Workspace.Workflow;
+
+class Example
+{
+    static void Main(string[] args)
+    {
+        // Find your Account Sid and Auth Token at twilio.com/console
+        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+        const string authToken = "your_auth_token";
+        const string workspaceSid = "WSXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+        const string workflowSid = "WWXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+
+        TwilioClient.Init(accountSid, authToken);
+
+        var stats = WorkflowStatisticsResource.Fetch(workspaceSid, workflowSid);
+
+        var cumulativeStats = JObject.FromObject(stats.Cumulative);
+
+        Console.WriteLine(cumulativeStats["avg_task_acceptance_time"]);
+
+        Console.WriteLine(cumulativeStats["tasks_entered"]);
+
+        var taskByStatusStats = JObject.FromObject(stats.Realtime)["tasks_by_status"]
```

Comment:
The attribute you're reading is `"task_by_status"`, based on that it makes sense to use a variable `taskByStatus` to hold this value. What do you think?